worker.py

```diff
@@ -890,6 +899,57 @@ def bag_full(self):
                     self.logger.warning("Failed to apply incubator {} on {}, code: {}".format(
                         inc.get('id', 0), egg.get('id', 0), ret))
 
+    async def handle_captcha(self, responses):
+        if not config.CAPTCHA_KEY:
+            self.logger.error("CAPTCHA Key is not set! this should not happen")
+            raise CaptchaException
+
+        if self.num_captchas >= config.CAPTCHA_RETRIES:
+            self.logger.error("User {} encountered a CAPTCHA to often, removing".format(self.username))
+            raise CaptchaException
+
+        self.num_captchas += 1
+
+        url = responses.get('CHECK_CHALLENGE', {}).get('challenge_url', ' ')
+        site_key = '6LeeTScTAAAAADqvhqVMhPpr_vB9D364Ia-1dSgK'
+        s = requests.Session()
```

Comment:
I may be able to get around to that, though I probably won't block this PR over that. I'm mainly waiting to hear test results.